gax/test/README.md

```diff
@@ -0,0 +1,92 @@
+# Tests for gax-nodejs
+
+## [browser-test](./browser-test/)
+### About
+End-to-end karma tests that run against the GAPIC [showcase server](#showcase-server)
+
+### Maintenance
+The [showcase-server](#showcase-server) should periodically be updated to the newest version of gapic-showcase. See [showcase server maintenance info](#maintenance-2) for more details. 
+
+## [fixtures](./fixtures)
+Fixtures for the [unit tests](#unit). These are not autogenerated and can be manually modified as needed.
+
+## [showcase-echo-client](./showcase-echo-client/)
+### About
+This name of this directory is a misnomer these days, as it now contains generated protos and clients for two of the [gapic-showcase](https://github.com/googleapis/gapic-showcase) clients: Echo and Sequence. These clients are used in the [test application](#test-application). The [Echo client](./showcase-echo-client/protos/google/showcase/v1beta1/echo.proto) showcases the four main types of GAPIC calls, and the [Sequence client](./showcase-echo-client/protos/google/showcase/v1beta1/sequence.proto) gives users the ability to define a sequence of either unary or server streaming responses from the server, enabling thorough testing of retry behavior.
+
+### Maintenance
+Currently, the process to update the showcase echo client is entirely manual. It involves two steps: [manually regenerating](#regenerate-the-clients) the files found in the directory and manually bumping the version of GAPIC showcase in the [showcase server](#showcase-server).
+
+#### Regenerate the clients
+The following steps will regenerate new Echo and Sequence clients from the latest version of GAPIC showcase.
+
+**Prerequsities**
```

Comment:
looks like a typo to me but I can never type this word correctly anyway

Reply:
LOL good catch